### DISCUSSION TITLE
Include non-JSON request bodies in parsed requests for triggers and endpoints

### DISCUSSION THREAD

**Discussion by LewisMcReu** [directus/directus#23164] [Feature Requests]
### Describe the Improvement

Directus does not parse any request body of type application/url-form-encoded in webhook triggers or on custom endpoints.
Considering as an end user you don't always control the content sent to your webhooks, it would be a simple improvement to still include request bodies that can't be parsed to JSON as a string.

**Comment by paulrudy** [comment]
This would be very helpful, for example with web form submissions.
